src/MySqlConnector/MySql.Data.MySqlClient/MySqlConnectionStringBuilder.cs

```diff
@@ -347,6 +355,10 @@ static MySqlConnectionStringOption()
 				keys: new[] { "SSL Mode", "SslMode" },
 				defaultValue: MySqlSslMode.Preferred));
 
+			AddOption(SslProtocols = new MySqlConnectionStringOption<SslProtocols>(
+				keys: new[] { "SSL Protocols", "SslProtocols" },
+				defaultValue: System.Security.Authentication.SslProtocols.Tls | System.Security.Authentication.SslProtocols.Tls11));
```

Comment:
This should be as secure as possible by default, and include `Tls12` on Linux

Reply:
Windows, or OS where client runs, is not important.  How server was compiled is important.

Reply:
why on non-Windows? Client OS is not important, it is not where the bug comes from.
Server OS is somewhat-important . On Linux, distributions always use openssl, but the tarballs  usually use Yassl. Windows was traditionally compiled with unpatched Yassl containing the bug, although MySQL Enterprise use openssl, recent MariaDB fixed the yassl negotiation bug, and  not-yet-released MySQL 8.0 ditched Yassl in its packaging.

Reply:
See https://github.com/mysql-net/MySqlConnector/pull/450#issuecomment-372850875